.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Prerequisites
+
+- [ ] I have written a descriptive pull-request title
+- [ ] I have verified that there are no overlapping [pull-requests](https://github.com/NancyFx/Nancy/pulls) open
+- [ ] I have verified made sure that I am following the Nancy [code style guidelines](https://github.com/NancyFx/Nancy/blob/45238076ad0b7f6ecabd6bae8469e30458d02efe/CONTRIBUTING.md#style-guidelines)
+- [ ] I have provided test coverage for your change (where applicable)
+
+### Description
+<!--
+[A description of the changes proposed in the pull-request]

```

Comment:
Can we split this comment in two, so it's easier to delete this line, while still keeping the "Thanks for contributing to Nancy!" comment in balance? Also, probably remove the wrapping `[]`